src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceTests.cs

```diff
@@ -503,10 +503,10 @@ public void AllListenersCheckCatchupList()
         [OuterLoop]
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process))] // [ActiveIssue(35539)]
         [InlineData(100, 100)] // run multiple times to stress it further
-        [InlineData(100, 100)]
-        [InlineData(100, 100)]
-        [InlineData(100, 100)]
-        [InlineData(100, 100)]
+        [InlineData(100, 101)]
+        [InlineData(100, 102)]
+        [InlineData(100, 103)]
+        [InlineData(100, 104)]
```

Comment:
I'm not sure if it xunit actually removed the duplicate inlinedata during execution. If it did then this will change how often we execute the test but I guess that's not an issue.

Reply:
> I'm not sure if it xunit actually removed the duplicate inlinedata during execution

It did/does not.